### ISSUE TITLE
This is not an issue

### ISSUE THREAD

**Issue by aleGuardiola** [CooperLawrence/scorbot-controller#1]
Hello man I want to make a similar library but to controll Eru2 but I cant find any documentation about it can you help me on it. A i want to make the library using C#, I looked in your code and I understand everything but I want to use the x, y and z axis to move the robot.

**Comment by CooperLawrence** [comment]
If you have the original program that came with it, I'd recommend decoding the outputs it sends to the arm's control board (I assume it uses RS-232 like the Scorbot ER 3). From there you can use any language with a serial library to control the arm.

**Comment by aleGuardiola** [comment]
Yes I have the original software but how i can decode the output that it send to the arm?. Do you have any suggestion?

**Comment by CooperLawrence** [comment]
If you have a computer with two serial ports, try connecting them together. Set the program to write on one of them, and listen to what's being received on the other with some serial terminal. If you don't have a computer with serial ports, you may be able to find some software to simulate them.
